File: core/src/main/java/com/example/app/movie/type/MovieRating.java
package com.example.app.movie.type;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum MovieRating {
    ALL_AGES("전체관람가"),
    TWELVE_ABOVE("12세 이상 관람가"),
    FIFTEEN_ABOVE("15세 이상 관람가"),
    NO_MINORS("청소년 관람불가"),
    RESTRICTED("제한상영가");

    private final String description;
}

File: cinema/cinema-domain/src/main/kotlin/com/study/cinema/domain/movie/MovieRating.kt
package com.study.cinema.domain.movie

enum class MovieRating(val description: String) {
    G("전체 관람가"),
    PG_13("12세 이상 관람가"),
    PG_15("15세 이상 관람가"),
    R("청소년 관람불가"),
    X("제한 상영가")
}